modules/setting/task.go

```diff
@@ -4,22 +4,15 @@
 
 package setting
 
-var (
-	// Task settings
-	Task = struct {
-		QueueType    string
-		QueueLength  int
-		QueueConnStr string
-	}{
-		QueueType:    ChannelQueueType,
-		QueueLength:  1000,
-		QueueConnStr: "addrs=127.0.0.1:6379 db=0",
-	}
-)
-
 func newTaskService() {
-	sec := Cfg.Section("task")
-	Task.QueueType = sec.Key("QUEUE_TYPE").MustString(ChannelQueueType)
-	Task.QueueLength = sec.Key("QUEUE_LENGTH").MustInt(1000)
-	Task.QueueConnStr = sec.Key("QUEUE_CONN_STR").MustString("addrs=127.0.0.1:6379 db=0")
+	taskSec := Cfg.Section("task")
+	queueTaskSec := Cfg.Section("queue.task")
+	switch taskSec.Key("QUEUE_TYPE").MustString(ChannelQueueType) {
+	case ChannelQueueType:
+		queueTaskSec.Key("TYPE").MustString("persistable-channel")
```

Comment:
Where does these values go?

Reply:
This is here to support the old task configuration and maps onto the queue configuration.

If the user configures a queue.task section then those values will override these task configuration sections.